.github/scality-storage-deployment/docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+services:
+  s3:
+    profiles: ['s3']
+    image: ${CLOUDSERVER_IMAGE}
```

Comment:
Can you a default value for the image, it's annoying having to set an extra env value when you're trying to run things locally. 

Not 100% sure about the syntax but something around `image: ${CLOUDSERVER_IMAGE:-ghcr.io/scality/cloudserver:my-version}` should do the trick.

Reply:
Its not supported locally, locally it would be using normal cloudserver.

Reply:
Reference ticket: https://scality.atlassian.net/browse/S3CSI-12

Reply:
I actually added this in [this commit](https://github.com/scality/mountpoint-s3-csi-driver/pull/13/commits/fbb7aa6e217c6b7c9f9ee0e0a13f0efff871745e)
But I didn't add dependabot way of doimgn things yet.
Thanks :)